cypress/integration/Checkbox/Displays_error.feature

```diff
@@ -0,0 +1,6 @@
+Feature: The Checkbox field displays an error when invalid
+
+    Scenario: The user submits a form with an unchecked required checkbox
+        Given an unchecked Checkbox is rendered
```

Comment:
I'd expect this to read `Given an unchecked required Checkbox is rendered`

Reply:
Actually, that makes me think. How can a checkbox be required? If the only valid state for a required checkbox is to be checked, then the user doesn't really have an option right? Or am I missing something?

Reply:
This was discussed on Slack:
- a valid use-case could be a "agree to terms and conditions" type of scenario
- a slightly different type of validator (i.e. requireTrue) might be more appropriate for this type of context, but that discussion is out of scope for the current PR

I will resolve this conversation